kernel_gateway/gatewayapp.py

```diff
@@ -152,9 +153,16 @@ def _default_kernel_name_default(self):
     def _list_kernels_default(self):
         return os.getenv(self.list_kernels_env, 'False') == 'True'
 
+    api_env = 'KG_API'
+    api = Unicode(config=True,
+        help='Designates that the kernel gateway will serve APIs from this notebook (KG_API env var)'
```

Comment:
Is there a choices traitlet? If so, this should use it. Either way, the currently supported options should be "jupyter-websocket" and "notebook-http". Anything else should be an error. The help string needs to reflect the available options if you can't define a set of choices in a traitlet.